### ISSUE TITLE
Simple example?

### ISSUE THREAD

**Issue by Ajusa** [yglukhov/rod#183]
Hey there!

I have been looking for something like this... Well for a while. Are there any simple examples available for new people (like me) to look at? A classic 3d example would be a cube rotating.

The editor was just too much code for me to understand the framework.

**Comment by hcorion** [comment]
This project: https://github.com/yglukhov/minecraft uses rod and nimx.

**Comment by yglukhov** [comment]
True, but it is not a good example of usage, just a quick and hacky port.

**Comment by Ajusa** [comment]
Yeah...
I would love to see examples. I understand that writing documentation as of now would be too much, compared to a few examples.

**Comment by r3d9u11** [comment]
@yglukhov Hi. Is there still no examples/docs for this engine?
